src/PresentationalComponents/OverviewDashbar/OverviewDashbar.js

```diff
@@ -0,0 +1,142 @@
+import { useGetPathwaysQuery } from '../../Services/Pathways';
+import { workloadQueryBuilder } from '../Common/Tables';
+import MessageState from '../MessageState/MessageState';
+import Loading from '../Loading/Loading';
+import messages from '../../Messages';
+import {
+  PATHWAYS,
+  INCIDENTS,
+  IMPORTANT_RECOMMENDATIONS,
+  CRITICAL_RECOMMENDATIONS,
+  IMPORTANT_TAG,
+  CRITICAL_TAG,
+  SEVERITY_MAP,
+  RECOMMENDATIONS_TAB,
+  PATHWAYS_TAB,
+} from '../../AppConstants';
+
+const getWrongTitleErrorMessage = (nameOfComponent) => {
+  return `Error! ${nameOfComponent} was provided with an invalid title. Valid titles are: ${
+    nameOfComponent === 'applyFiltersByTitle' ? '' : `'${PATHWAYS}', `
+  }'${INCIDENTS}', '${IMPORTANT_RECOMMENDATIONS}', '${CRITICAL_RECOMMENDATIONS}'`;
+};
+
+/*****************************************/
+const OverviewDashbar = ({ changeTab }) => {
+  const { offset } = useSelector(({ filters: { pathState } }) => pathState);
+  const selectedTags = useSelector(({ filters }) => filters.selectedTags);
+  const workloads = useSelector(({ filters }) => filters.workloads);
+  const SID = useSelector(({ filters }) => filters.SID);
+
+  const dispatch = useDispatch();
+  const { formatMessage } = useIntl();
+
+  const options = {
+    ...(selectedTags?.length > 0 ? { tags: selectedTags.join(',') } : {}),
+    ...(workloads ? workloadQueryBuilder(workloads, SID) : {}),
+  };
+  const { data, isLoading, isFetching, isError } = useGetPathwaysQuery({
+    sort: '-recommendation_level',
+    offset,
+    limit: 3,
+    ...options,
+  });
+
+  // the initial filters state in recommendations table
+  const { recState: defaultFilters } = filtersInitialState;
+  // This function is used to apply filters to the recommendations table
+  // It resets the values to be the default values, and then adds the values that are passed to it in Addedfilters
+  const applyFilters = (Addedfilters) => {
+    dispatch(updateRecFilters({ ...defaultFilters, ...Addedfilters }));
+  };
+
+  // this function is used to apply filters to the recommendations table based on the title of the card that was clicked
+  // also, it changes the tab to the recommendations tab (only if the title matches one of the recommendations titles: critical, important, or incidents)
+  const applyFiltersByTitle = (title) => {
+    let needToChangeTab = true;
+    switch (title) {
+      case INCIDENTS:
+        applyFilters({ incident: true });
+        break;
+      case CRITICAL_RECOMMENDATIONS:
+        applyFilters({ total_risk: SEVERITY_MAP[CRITICAL_TAG] });
+        break;
+      case IMPORTANT_RECOMMENDATIONS:
+        applyFilters({ total_risk: SEVERITY_MAP[IMPORTANT_TAG] });
+        break;
+      default:
+        needToChangeTab = false;
+        console.log(getWrongTitleErrorMessage('applyFiltersByTitle'));
+        break;
+    }
+    if (needToChangeTab) {
+      changeTab(RECOMMENDATIONS_TAB);
```

Comment:
It should call `changeTab` only for the specified cases (`INCIDENTS`, `CRITICAL_RECOMMENDATIONS`, `IMPORTANT_RECOMMENDATIONS`)
In any other case, we would not want to change the Tab (using this function)

Do you think it would be better to just call `changeTab` inside the cases? and remove `needToChangeTab` completely ?

Reply:
@LiorKGOW Yeah, let's remove the `needToChangeTab` variable. I already misunderstood when it is supposed to be called. :D

Reply:
> @LiorKGOW Yeah, let's remove the `needToChangeTab` variable. I already misunderstood when it is supposed to be called. :D

😅 I will remove it 👍🏼

Reply:
`needToChangeTab` variable was removed 👍🏼